tests/python/contrib/test_ethosn/test_leaky_relu.py

```diff
@@ -55,9 +56,12 @@ def test_leaky_relu(dtype, shape, alpha):
     iinfo = np.iinfo(dtype)
     zp_min = iinfo.min
     zp_max = iinfo.max
-    input_zp = zp_min + 120
+    if ethosn_api_version() == "3.2.0":
+        input_zp = zp_min + 128
+    else:
+        input_zp = zp_min + 120
```

Comment:
Probably not worth the conditional, assuming `zp_min + 120` works for both the previous and new version?

Reply:
I thought after this is fixed in 23.02, it will be worth having both the cases as a test.

Reply:
Ah okay make sense, let's leave it for now then :)